tests/fv3_conf/cpld_bmark_run.IN

```diff
@@ -8,19 +8,15 @@ fi
 
 ICERES="${OCNRES:0:1}.${OCNRES:1}"
 
-if [[ ${FRAC_GRID_INPUT} = .F. ]]; then
-  FV3_IC=@[INPUTDATA_ROOT_BMIC]/${SYEAR}${SMONTH}${SDAY}${SHOUR}/gfs/@[ATMRES]/INPUT
-elif [[ @[NPZ] == '127' ]]; then
-  FV3_IC=FV3_input_frac/BM_IC/${SYEAR}${SMONTH}${SDAY}${SHOUR}/gfs/@[ATMRES]_L@[NPZ]/INPUT
-else
-  FV3_IC=FV3_input_frac/BM_IC/${SYEAR}${SMONTH}${SDAY}${SHOUR}/gfs/@[ATMRES]/INPUT
-fi
+FV3_IC=@[INPUTDATA_ROOT_BMIC]/${SYEAR}${SMONTH}${SDAY}${SHOUR}/gfs_p6/@[ATMRES]_L@[NPZ]/INPUT
+MOM_IC=@[INPUTDATA_ROOT_BMIC]/${SYEAR}${SMONTH}${SDAY}${SHOUR}/mom6_da
+ICE_IC=@[INPUTDATA_ROOT_BMIC]/${SYEAR}${SMONTH}${SDAY}${SHOUR}/cpc
 #unused unless 35D run is active
-WW3_IC=FV3_input_frac/BM_IC/${SYEAR}${SMONTH}${SDAY}${SHOUR}/wav/gwes_30m
+WW3_IC=@[INPUTDATA_ROOT_BMIC]/${SYEAR}${SMONTH}${SDAY}${SHOUR}/wav_p6/gwes_30m
```

Comment:
The ICs that are needed for the develop branch are not the "p6" ICs.  On one hand, I don't think its important but on the other hand, I don't want people to think it's p6 when it's not.  Also these ICs will occasionally change.  One thing we could do is put the date of the WW3 commit that it corresponds to?  I think it's good to have this subfolder though because we can add another folder and then just update this instead of having to do a completely new INPUTDATA_ROOT_BMIC unless that's what we want every time the WW3 IC has to be updated.

Reply:
These were copied from the existing FV3_input_frac/BM_IC wave inputs. I realize they are probably not what was used in P6. I actually thought about leaving the entire wave/p6 directory out because they are used only for the 35d runs, and we will not be making any P6 35d runs from the RTs. However, we're still carrying the  ``cpld_bmark_wave_v16_35d`` which requires something for the WW3 ICs. I think maybe the best solution is to remove that 35d script and the wav_p6 directory also. We are still carrying the v16 RTs, so we need the ATM ICs.

The wave ICs will not be updated each time the WW3 ICs are updated. They are used only for the 35d runs, so would not be updated until the next Prototype was being set up. At that time, we'll probably have new ATM ICs also, from which the WW3 ICs can be generated. I don't think we need a dated WW3 IC sub-directory.

Reply:
Following the discussion at this morning's tag-up, I will be removing all but the **2013040100** P6 ATM ICs. These ICs are required to run the current v16 benchmark RTs. No wave ICs are required. I will also remove the non-p7 35d scripts.